### PR TITLE
Improve error handling of stencil.renderToString()

### PR DESCRIPTION
**Pull request by BrunnerLivio**

### CODE REVIEW

File: gatsby-node.js [hunk 1]
```diff
@@ -1,19 +1,35 @@
...
         const html = fs.readFileSync(file, "utf8");
         const result = await stencil.renderToString(html, {
-          prettyHtml: true
+          prettyHtml: true,
         });
+
+        if (result.html === null) {
+          const errors = getStencilErrors();
+          if (errors.length) {
+            throw new Error(`${errors[0].header}: ${errors[0].messageText}`);
```
Comment (BrunnerLivio) [inline gatsby-node.js:26]: Not the best approach -- it only takes the first error. Suggestions?

**Comment by jonearley** [comment]
@BrunnerLivio -- this looks like a good change. Could you explain a bit more why you're looking to add this? Did you run into an issue that you could only understand by adding Stencil errors?

**Comment by BrunnerLivio** [comment]
@jonearley I honestly have not had time to investigate further what’s the problem with my setup (was not been able to fix it as of now). I thought I’d submit this PR, since it helped me to debug it and have more verbose feedback.

**Comment by timomayer** [comment]
can you please merge this PR? while not beeing optimal its far better then having nothing and "null" in your static pages.

i would welcome to have this released :)

**Comment by BrunnerLivio** [comment]
Closing in favor of #3

**Pull request closed without merging**